app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: :User
+  belongs_to :post
+end
```

Comment:
- [ ] Kindly add a method in Comment model  that updates the comments counter for a post.

eg:
```ruby
def update_comments_counter
  post.increment!(:comments_counter)
end
```